workflow/exec_workflow.sh

```diff
@@ -14,7 +14,6 @@ workflow_name=$2
 indexfile="workflow_index-$resource_name-$workflow_name.tmp"
 cmdsfile="workflow_cmds-$resource_name-$workflow_name.tmp"
 
-WORKFLOW_INDEX=$(cat $indexfile | head -n 1)
-# echo "executing command $WORKFLOW_INDEX: '$( head -n $WORKFLOW_INDEX $cmdsfile | tail -n 1 )'"
-$(head -n $WORKFLOW_INDEX $cmdsfile | tail -n 1)
+WORKFLOW_INDEX=$(head -n 1 <"$indexfile")
```

Comment:
I noticed if I rewind then replay immediately, I get an error (since index is `0`):
```sh
# reset
Running cmd: /Users/milas/dev/tilt-extensions/workflow/rewind.sh test (Tiltfile)
resetting...

# immediate replay
Running cmd: /Users/milas/dev/tilt-extensions/workflow/replay.sh test (Tiltfile)
head: illegal line count -- 0
/Users/milas/dev/tilt-extensions/workflow/replay.sh test (Tiltfile) exited with exit code 1
```

I think maybe just exiting with a "no step to replay" type message if `$indexfile` is `0` would be sufficient.

Reply:
Done. Thank you!